app/controllers/api/articles_controller.rb

```diff
@@ -17,9 +19,21 @@ def index
 
   def show
     article = Article.find(params[:id])
-
     render json: article, serializer: ArticlesShowSerializer
   rescue ActiveRecord::RecordNotFound => e
     render json: { error_message: "Article does not exist" }, status: 404
   end
+
+  def create
+    unless current_user.role == "journalist"
+      render json: { error_message: "You dont have access" }, status: 403 and return
+    end
+
+    article = current_user.articles.create(params[:article].permit(:title, :teaser, :body, :category))
```

Comment:
Not a blocker, but a good practice is to extract this to a `private` method

```ruby
def article_params
   params[:article].permit(:title, :teaser, :body, :category)
end
```

and then use it:

```ruby
article = current_user.articles.create(article_params)
```

Reply:
Good point x2!